app/experimenter/settings.py

```diff
@@ -91,6 +91,8 @@
 
 WSGI_APPLICATION = "experimenter.wsgi.application"
 
+# https://docs.djangoproject.com/en/2.1/ref/settings/#internal-ips
+INTERNAL_IPS = config("INTERNAL_IPS", cast=Csv(), default="127.0.0.1")
```

Comment:
Is this related to Google Analytics?

Reply:
It's needed for the `django.template.context_processors.debug` context processor.